### PR TITLE
fix: remove direct install from setup file

### PR DESCRIPTION
**Pull request by unparalleled-js**
### What I did

Thanks @fubuloubu for the help with this one and the fast turn-around.
This should fix publishing 🤞🏻 

fixes: APE-550

### How I did it

* Comment out myst parser in setup.py
* Install from github in the docs CI action

### How to verify it

We try releasing again!

### Checklist

- [ ] All changes are completed
- [ ] New test cases have been added
- [ ] Documentation has been updated

### CODE REVIEW

**Review by NotPeopling2day — DISMISSED** [review]

**Review by fubuloubu — APPROVED** [review]

**Review by NotPeopling2day — APPROVED** [review]

File: .github/workflows/prtitle.yaml [hunk 1]
```diff
@@ -22,7 +22,7 @@ jobs:
         - name: Install Dependencies
           run: |
             python -m pip install --upgrade pip
-            pip install .[dev]
+            pip install "commitizen>=2.40,<2.41"
```
Comment (unparalleled-js) [inline .github/workflows/prtitle.yaml:25]: The action runs much faster this way and saves action minutes.
It only needs the one tiny tool to do this job! It does not need Ape installed.

**Comment by unparalleled-js** [comment]
The Pr-title action somehow only uses either the yaml from when the PR was first opened or the yaml from main, can't tell.
It always fails even though I am 99.9% sure I fixed the issue.

I say we merge this and I can open a dummy PR next to show that it is indeed fixed.

File: .github/workflows/prtitle.yaml (hunk 1 above)
Comment (fubuloubu) [reply .github/workflows/prtitle.yaml:25]: Yes this is true! I wanted it to source the same version of commitizen from setup.py, only reason I changed it that way

**Review by fubuloubu — DISMISSED** [review]

File: .github/workflows/prtitle.yaml (hunk 1 above)
Comment (unparalleled-js) [reply .github/workflows/prtitle.yaml:25]: Should we go back? I am good with other. Pros and cons!

Comment (fubuloubu) [reply .github/workflows/prtitle.yaml:25]: so like if it could just read that line directly from `setup.py` that would be best

Comment (unparalleled-js) [reply .github/workflows/prtitle.yaml]: [f35f8eb](https://github.com/ApeWorX/ape/pull/1267/commits/f35f8ebe9d227a906d4b245c151e15d573dcaf3e)

File: setup.py [hunk 2]
```diff
@@ -47,15 +47,16 @@
         "twine==3.8.0",  # Package upload tool
     ],
     "dev": [
-        "commitizen>=2.19,<2.20",  # Manage commits and publishing releases
+        # commitizen: Manage commits and publishing releases
+        (here / "cz-requirement.txt").read_text().strip(),
```
Comment (unparalleled-js) [inline setup.py:51]: @fubuloubu Now, `setup.py` and the actions that use CZ share the same source of truth.

**Review by fubuloubu — APPROVED** [review]

**Review by fubuloubu — APPROVED** [review]

**Review by NotPeopling2day — APPROVED** [review]

**Pull request merged**